### PR TITLE
feat: update channel

### PR DESCRIPTION
**Pull request by rolznz**
Users can now update base forwarding fees on public channels. A high value can be set to prevent routing.

image

image

### CODE REVIEW

File: lnclient/ldk/ldk.go [hunk 1]
```diff
@@ -849,8 +860,13 @@ func (ls *LDKService) OpenChannel(ctx context.Context, openChannelRequest *lncli
 	ldkEventSubscription := ls.ldkEventBroadcaster.Subscribe()
 	defer ls.ldkEventBroadcaster.CancelSubscription(ldkEventSubscription)
 
+	channelConfig := ldk_node.NewChannelConfig()
+
+	// TODO: consider setting a super-high forwarding fee by default to disable unwanted routing
```
Comment (rolznz) [inline lnclient/ldk/ldk.go:865]: Should we enable this?

Comment (bumi) [reply lnclient/ldk/ldk.go:865]: yes, I think we should for now.

File: lnclient/lnd/lnd.go [hunk 2]
```diff
@@ -727,6 +727,10 @@ func (svc *LNDService) GetNetworkGraph(nodeIds []string) (lnclient.NetworkGraphR
 
 func (svc *LNDService) UpdateLastWalletSyncRequest() {}
 
+func (svc *LNDService) UpdateChannel(ctx context.Context, updateChannelRequest *lnclient.UpdateChannelRequest) error {
```
Comment (bumi) [inline lnclient/lnd/lnd.go:730]: @im-adithya can you implement this?

**Review by bumi — APPROVED** [review]
utACK!

File: lnclient/lnd/lnd.go (hunk 2 above)
Comment (im-adithya) [reply lnclient/lnd/lnd.go:730]: Will do 👍

Comment (im-adithya) [reply lnclient/lnd/lnd.go:730]: Opened #459

**Pull request merged**